pkg/inventory/inventory-client_test.go

```diff
@@ -104,15 +105,13 @@ func TestGetClusterInventoryInfo(t *testing.T) {
 			if !tc.isError && err != nil {
```

Comment:
This can just be if yes require.Error, else require.NoError

Reply:
done